packages/nimble-components/src/table/tests/table.pageobject.ts

```diff
@@ -16,8 +17,8 @@ export class TablePageObject<T extends TableRecord> {
         return headers.length;
     }
 
-    public getRenderedHeaderContent(columnIndex: number): string {
-        const headers = this.tableElement.shadowRoot!.querySelectorAll(
+    public getHeaderContent(columnIndex: number): Node | undefined | null {
```

Comment:
We have learned more towards avoiding leaking null values. We should try and push this to return `Node | undefined`

Reply:
Yeah that didn't need to be there...removed.